README.md

```diff
@@ -1613,6 +1614,33 @@ ANSWERS.addComponent('GeoLocationFilter', {
 });
 ```
 
+## Applied Filters Component
+
+The Applied Filters Component displays your currently applied filters as a row of text tags, labeled
+by filter display value. If the "removable" config option is set to true, these text tags will instead
+be "removable filters", which, when clicked, will remove the clicked filter from the search.
+Only intended for vertical pages.
+
+```js
+ANSWERS.addComponent('AppliedFilters', {
+  container: '.applied-filters-container',
+  // The vertical key of your search, will default to the vertical key specified in the search config.
```

Comment:
Could these prop comments be prefixed with "Required," or "Optional"?

Reply:
Sure, they're all optional (except for container) but it's better to be consistent